tests/fuzz/src/read_network-fuzzer.cc

```diff
@@ -38,7 +38,9 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
     try {
         InferenceEngine::Core ie;
         InferenceEngine::CNNNetwork network = ie.ReadNetwork(net, weights_blob);
-    } catch (const InferenceEngine::details::InferenceEngineException& error) {
+    } catch (const InferenceEngine::details::InferenceEngineException&) {
+        return 0;  // fail gracefully on expected exceptions
+    } catch (const std::exception&) {
         return 0;  // fail gracefully on expected exceptions
```

Comment:
```suggestion
    } catch (...) {
        return 0;  // fail gracefully on expected exceptions
```

Reply:
@ilyachur this fuzz test is also designed to verify IE does not throw unexpected exceptions. IE API can only throw std::except and its derivatives as defined per [samples implementation](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/samples/hello_classification/main.cpp#L140)